website/docs/r/muting_schedule.html.markdown

```diff
@@ -29,125 +28,76 @@ resource "sumologic_muting_schedule" "muting_schedule" {
-	monitor {
-		ids = ["0000000000200B92"]
-	  }
-	schedule  {
-		timezone = "America/Los_Angeles"
-		start_date = "2023-08-05"
-		start_time = "00:00"
-		duration = 30
-    	rrule = "FREQ=DAILY;INTERVAL=1;BYHOUR=9,10"
-	}
-}
-```
-
-## Example Daily Muting Schedule With Group Supporting 
-
-```hcl
-resource "sumologic_muting_schedule" "muting_schedule" {
-  name = "Muting Schedule For one time"
-  description = "This is an example for muting schedule with rrule and specific monitor id and notification group key and values"
-	type = "MutingSchedulesLibraryMutingSchedule"
-	content_type = "MutingSchedule"
-	monitor {
-		ids = ["0000000000200B92"]
-	  }
-    notification_groups {
-        group_key = "_sources"
-        group_values =["localhost","127.0.0.1"]
-      }
-	schedule  {
-		timezone = "America/Los_Angeles"
-		start_date = "2023-08-05"
-		start_time = "00:00"
-		duration = 30
-    	rrule = "FREQ=DAILY;INTERVAL=1;BYHOUR=9,10"
-	}
-}
-```
-
 ## Argument reference
 
 The following arguments are supported:
 
-- `type` - (Optional) The type of object model. Valid value:
-  - `MutingSchedulesLibraryMutingSchedule`
-- `name` - (Required) The name of the muting schedule. The name must be alphanumeric.
-- `description` - (Optional) The description of the muting schedule.
-- `content_type` - (Optional) The type of the content object. Valid value:
-  - `MutingSchedule`
-- `monitor` - (Optional) The monitors which need to put in the muting schedule. see `monitor_scope`:
-- `schedule` - (Required) The schedule information. see `schedule`.
-- `notification_groups` -(Optional) The muting schedule group supporting key and values. see `notification_group`
+- `type` - (Required) The type of object model. Valid value: `MutingSchedulesLibraryMutingSchedule`
+- `name` - (Required) Name of the muting schedule.
+- `description` - (Optional) Description of the muting schedule.
+- `content_type` - (Optional) The type of the content object. Valid value: `MutingSchedule`
+- `monitor` - (Optional) Monitor scope that the schedule applies to. See `Monitor Scope` for more details.
+- `schedule` - (Required) Schedule definition. See `Schedule Definition` for more details.
+- `notification_groups` - (Optional) Alert group scope that the schedule applies to. See `Group Scope` for more details.
 
-#### schedule
+#### Schedule Definition
   - `timezone` - (Required) Time zone for the schedule per
             [IANA Time Zone Database](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones#List).
   - `start_date` - (Required) Schedule start date in the format of `yyyy-mm-dd`
   - `start_time` - (Required) Schedule start time in the format of `hh:mm`
   - `duration` - (Required) Duration of the muting in minutes
-  - `rrule` - (Optional) RRule (Recurrence Rule) Below are some examples of how to represent recurring events using the RRULE format:
-  A rule occurring on the third Sunday of April would be as follows: `FREQ=YEARLY;BYMONTH=4;BYDAY=SU;BYSETPOS=3`
```

Comment:
Having these examples can be helpful. Any reason to remove them?

Reply:
I think this seems redundant given that we already provided the link (https://freetools.textmagic.com/rrule-generator) and we already have some examples on the same page above e.g. https://github.com/SumoLogic/terraform-provider-sumologic/pull/601/files#diff-647440352a7c9c50a68f41c831b6e35221582ea5429f951464e1ac25e925fadeR70 .